.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Description of the issue**
+
+**How to reproduce the issue**
+
+\`\`\`
+
+put code in code blocks
```

Comment:
nit: "put code in code blocks *like this*" is a little more concrete (it implicitly explains what a "code block" is) and sounds even less formal